setup.py

```diff
@@ -40,5 +40,11 @@
         "logilab-common == 0.62.0",
         "pep8 == 1.5.7"
     ],
+    extras_require = {
+        'dev':  [
+            'twisted==13.2.0',
```

Comment:
I think this is too old a version of Twisted; we should be encouraging people to use the latest.